client/app/assets/less/inc/visualizations/sankey.less

```diff
@@ -12,3 +12,8 @@
   stroke: #000;
   stroke-opacity: .2;
 }
+
+.sankey-visualization-container {
+  height: 300px;
```

Comment:
Does this make Sankey fixed height?

Reply:
Here (and in other components) - it's the default height (when dashboard is in mobile mode or when auto-height enabled) - just moved from component's config.

Reply:
👌 this is the same for embeds, right?

Reply:
Yes, sure

Reply:
And for preview in editors too